src/Dict.cc

```diff
@@ -430,72 +433,75 @@ int Dictionary::Next(int position) const
 	}
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////
-//Debugging
+// Debugging
 ///////////////////////////////////////////////////////////////////////////////////////////////////////
-#define DUMPIF(f) if(f) Dump(1)
+#define DUMPIF(f)                                                                                  \
+	if ( f )                                                                                       \
+	Dump(1)
```

Comment:
This should be indented a step, though the whole change is sort of pointless.

Reply:
indentation seems wrong yes; re/ the change as a whole: it is what it is :)

Reply:
Um, isn't it a breaking change, since it requires `\` at the end of the first two lines?

Reply:
They're just not showing up because clang-format likes to send those backslashes way over to the right, and github is adding a scrollbar to the pane.